tutorials/deploy-mean-app-mongodb-replication.md

```diff
@@ -0,0 +1,205 @@
+
+## Connect the MEAN application to the Bitnami MongoDB with Replication cluster
+
+Before proceeding, confirm that the Bitnami MEAN instance and the Bitnami MongoDB with Replication cluster instances are on the same VPC network. You can check this by visiting each instance's detail page from the [Deployment Manager](https://console.cloud.google.com/dm/deployments) and viewing the network name in the `Network interfaces` section. If the Bitnami MEAN instance and the Bitnami MongoDB with Replication cluster instances are not on the same network, [configure VPC peering between the two networks](https://cloud.google.com/vpc/docs/vpc-peering) before proceeding.
+
+Reconfigure the MEAN application to use the Bitnami MongoDB with Replication cluster:
+
+1. From the Bitnami MEAN instance, login to the Bitnami MongoDB with Replication cluster using the `mongo` command-line client. Use the internal IP address of the primary MongoDB node as the host IP address and enter the password obtained from the [Deployment Manager](https://console.cloud.google.com/dm/deployments) when prompted.
+
+        mongo --host XX.XX.XX.XX admin --username root -p
+
+1. Once logged in, create a new database and user account for the application:
+
+        use mydb
+        db.createUser(
+           {
+             user: "myuser",
+             pwd: "mypass",
+             roles: [ "readWrite" ]
+           }
+        )
+        exit
+
+1. Modify the application configuration to use the new Bitnami MongoDB with Replication cluster and credentials by modifying the `/opt/bitnami/apps/myapp/htdocs/config.json` file as shown below. Replace the `XX`, `YY` and `ZZ` placeholders with the internal IP addresses of all the nodes in the Bitnami MongoDB with Replication cluster, starting with the primary node.
+
+        {
+            "connectionString": "mongodb://myuser:mypass@XX.XX.XX.XX:27017,YY.YY.YY.YY:27017,ZZ.ZZ.ZZ.ZZ:27017/mydb?replicaSet=rs0",
+            ...
+        }
+
+1. Restart the Express server:
+
+        cd /opt/bitnami/apps/myapp/htdocs
+        forever restart server.js
+
+Browse to the public IP address of your Bitnami MEAN instance and confirm that it is working as expected. Note that since the MEAN application is now connected to a different MongoDB database, it will be necessary to re-register a user account before logging in to the example application.
+
+## Test service continuity (optional)
+
+In the Bitnami MongoDB with Replication template, one node in the cluster is designated as the primary node and receives all write operations, while other nodes are designated as secondary nodes which hold their own copies of the data set. In case the primary node fails, the secondary nodes and arbiters will elect a new primary automatically to ensure that writes continue without interruption. This provides redundancy and ensures minimal downtime for your application.
```

Comment:
Change "In case the primary node fails, the secondary nodes and arbiters will elect" to "If the primary node fails, the secondary nodes and arbiters elect"